buildSrc/src/main/kotlin/scripts/infrastructure.gradle.kts

```
@@ -40,7 +41,13 @@ tasks.register("runUnitTests") {
 
 tasks.register("runAcceptanceTests") {
     description = "Runs all Acceptance Tests in the connected device."
-    dependsOn(":app:connected${Default.BUILD_FLAVOR.capitalize()}DebugAndroidTest")
+    dependsOn(
+        ":app:connected${
+            Default.BUILD_FLAVOR.replaceFirstChar {
+                if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString()
+            }
```

suggestion: why not just make the first char a upper case 

```suggestion
            Default.BUILD_FLAVOR.replaceFirstChar(Char::titlecaseChar)
```

---

and this change my break Jenkins

---

could be let's try it